storage/innobase/os/os0file.cc

```diff
@@ -6203,9 +6203,15 @@ os_file_read_page(
 			}
 		}
 
+		char fdname[FN_REFLEN];
+		snprintf(fdname, sizeof fdname, "/proc/%d/fd/%d", getpid(), file);
+		char filename[FN_REFLEN];
+		my_readlink(filename, fdname, MYF(0));
```

Comment:
Any errors from my_readlink should be handled by gracefully degrading to the old output format

Reply:
Done